tpm2_pytss/tsskey.py

```diff
@@ -0,0 +1,20 @@
+"""
+SPDX-License-Identifier: BSD-3
+"""
+
+
+class TSSKey(object):
```

Comment:
TSS2PrivateKey or something to make it clear that its only for the private PEM files.

Reply:
I'll rename it to TSSPrivKey, as that is what it's called in tpm2-openssl